plugins/alert-handlers/snmp-alerts/src/org/apache/cloudstack/alert/snmp/SnmpHelper.java

```diff
@@ -65,6 +66,7 @@ private PDU createPDU(SnmpTrapInfo snmpTrapInfo) {
 
         int alertType = snmpTrapInfo.getAlertType() + 1;
         if (alertType > 0) {
+            trap.add(new VariableBinding(SnmpConstants.sysUpTime, new OctetString(new Date().toString())));
```

Comment:
@DaanHoogland Ok

I will use this  expression to replace now
sysUpTimeTicks = ManagementFactory.getRuntimeMXBean().getUptime() / 10;
trap.add(new VariableBinding(SnmpConstants.sysUpTime, new TimeTicks(sysUpTimeTicks)))